backend/src/routes/v2/model/postModel.ts

```diff
@@ -12,6 +12,7 @@ export const postModelSchema = z.object({
     name: z.string({
       required_error: 'Must specify model name',
```

Comment:
For consistency and as you are editing this file, could you also remove the other `required_error` fields?

Reply:
Done